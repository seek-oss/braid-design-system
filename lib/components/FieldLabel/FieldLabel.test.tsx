import React from 'react';
import { FieldLabel } from '..';
import { render } from 'react-testing-library';
import forEachTheme from '../../test/utils/forEachTheme';

describe('FieldLabel', () => {
  forEachTheme(({ ThemeProvider }) => {
    test('Default', () => {
      const { container } = render(
        <ThemeProvider>
          <FieldLabel>Label</FieldLabel>
        </ThemeProvider>
      );
      expect(container.firstChild).toMatchSnapshot();
    });
  });
});
