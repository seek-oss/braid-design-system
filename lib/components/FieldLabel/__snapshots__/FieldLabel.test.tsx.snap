// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FieldLabel Theme: jobStreet Default 1`] = `
<label
  class="Text__block atoms__fontFamily_text atoms__color_neutral atoms__fontSize_standard atoms__fontWeight_strong atoms__transform_standardText Box__root atoms__borderColor_standard"
>
  Label
</label>
`;

exports[`FieldLabel Theme: seekAnz Default 1`] = `
<label
  class="Text__block atoms__fontFamily_text atoms__color_neutral atoms__fontSize_standard atoms__fontWeight_strong atoms__transform_standardText Box__root atoms__borderColor_standard"
>
  Label
</label>
`;

exports[`FieldLabel Theme: seekAsia Default 1`] = `
<label
  class="Text__block atoms__fontFamily_text atoms__color_neutral atoms__fontSize_standard atoms__fontWeight_strong atoms__transform_standardText Box__root atoms__borderColor_standard"
>
  Label
</label>
`;

exports[`FieldLabel Theme: wireframe Default 1`] = `
<label
  class="Text__block atoms__fontFamily_text atoms__color_neutral atoms__fontSize_standard atoms__fontWeight_strong atoms__transform_standardText Box__root atoms__borderColor_standard"
>
  Label
</label>
`;
